docker-compose-tls.yml

```diff
@@ -61,32 +61,53 @@ services:
       - postgresql
       - elasticsearch
     environment:
-      - DB=postgresql
-      - DB_PORT=5432
-      - POSTGRES_USER=temporal
-      - POSTGRES_PWD=temporal
-      - POSTGRES_SEEDS=postgresql
-      - SQL_TLS=true
-      - DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development-sql.yaml
-      - ENABLE_ES=true
-      - ES_SEEDS=elasticsearch
-      - ES_VERSION=v7
-      - ES_SCHEME=https
-      - ES_USER=elastic
-      - ES_PWD=elastic
+      DB: postgresql
```

Comment:
Please don't change the yaml formatting of the environment block, it should be consistent with our other compose files.